examples/chart/teleport-cluster/values.schema.json

```diff
@@ -0,0 +1,289 @@
+                    "type": "string",
+                    "default": "teleport-gcp-credentials"
+                }
+            }
+        },
+        "highAvailability": {
+            "$id": "#/properties/highAvailability",
+            "type": "object",
+            "required": [
+                "replicaCount",
+                "requireAntiAffinity",
+                "certManager"
+            ],
+            "properties": {
+                "replicaCount": {
+                    "$id": "#/properties/highAvailability/properties/replicaCount",
+                    "type": "integer",
+                    "default": 1
+                },
+                "requireAntiAffinity": {
+                    "$id": "#/properties/highAvailability/properties/requireAntiAffinity",
+                    "type": "boolean",
+                    "default": false
+                },
+                "certManager": {
+                    "$id": "#/properties/highAvailability/properties/certManager",
+                    "type": "object",
+                    "required": [
+                        "enabled",
+                        "issuerName",
+                        "issuerKind"
+                    ],
+                    "properties": {
+                        "enabled": {
+                            "$id": "#/properties/highAvailability/properties/certManager/properties/enabled",
+                            "type": "boolean",
+                            "default": "false"
+                        },
+                        "issuerName": {
+                            "$id": "#/properties/highAvailability/properties/certManager/properties/issuerName",
+                            "type": "string",
+                            "default": ""
+                        },
+                        "issuerKind": {
+                            "$id": "#/properties/highAvailability/properties/certManager/properties/issuerKind",
+                            "type": "string",
+                            "default": "Issuer"
+                        }
+                    }
+                }
+            }
+        },
+        "image": {
+            "$id": "#/properties/image",
+            "type": "string",
+            "default": "quay.io/gravitational/teleport"
+        },
+        "enterpriseImage": {
+            "$id": "#/properties/enterpriseImage",
+            "type": "string",
+            "default": "quay.io/gravitational/teleport-ent"
+        },
+        "logLevel": {
+            "$id": "#/properties/logLevel",
+            "type": "string",
+            "enum": ["DEBUG", "INFO", "WARN", "WARNING", "ERROR"],
+            "default": "INFO"
+        },
+        "annotations": {
+            "$id": "#/properties/annotations",
+            "type": "object",
+            "required": [
+                "config",
+                "deployment",
+                "pod",
+                "service",
+                "serviceAccount"
```

Comment:
are all of these required?
a user should be able to set just .e.g `deployment` annotations without having to specify all of them, right?

Reply:
That's correct. See explanation [here](https://github.com/gravitational/teleport/pull/6344#discussion_r624196201) for context.